### DISCUSSION TITLE
just a friendly thank you

### DISCUSSION THREAD

**Discussion by matanox** [mkdocstrings/mkdocstrings#677] [General]
I just made my first few mkdocstrings enabled documentation pages for a subset of my python project's modules. I admit to not fully understanding how to link pages the mkdocs way as opposed to regular markdown links, but with mkdocstrings I could even embed a link from one function's docstring to another page and it is all very nice, beautiful and useful.

**Comment by pawamoy** [comment]
Thank you for the kind words 😊 I'm happy to hear that mkdocstrings is useful to you! If you'd like to get help on specific features of MkDocs or mkdocstrings, don't hesitate to open Q/A discussions in the relevant repositories 🙂 In particular, linking between pages should be very similar to usual Markdown linking. If something doesn't work as you would expect, let us know!
